server/cli/src/plugins.rs

```diff
@@ -75,22 +88,41 @@ pub(super) struct GenerateAndInstallPluginBundle {
     #[clap(long)]
     password: String,
 }
+// THe expected package.json format is as follows:
+// Front end: https://github.com/msupply-foundation/open-msupply-plugins/blob/81b78c31e5f938dd8b30783f7e3ee97555285f70/frontend/latest/package.json#L6-L14
+// Back end: https://github.com/msupply-foundation/open-msupply-plugins/blob/81b78c31e5f938dd8b30783f7e3ee97555285f70/backend/latest/package.json#L4-L10
+#[derive(Deserialize, Debug)]
```

Comment:
Using package.json now to find plugins, they have common property as per above comment,